install.sh

```diff
@@ -645,17 +612,17 @@ _start() {
 ###################################################
 _uninstall() {
     _print_center "justify" "Uninstalling.." "-"
-    __bak="source ${INFO_PATH}/google-drive-upload.binpath"
-    if _new_rc="$(sed "s|${__bak}||g" "${SHELL_RC}")" &&
-        printf "%s\n" "${_new_rc}" >| "${SHELL_RC}"; then
+    __bak="(.\|source) ${INFO_PATH}/google-drive-upload.binpath"
+    if _new_rc="$(sed -E "s|${__bak}||g" "${SHELL_RC}")" && printf "%s\n" "${_new_rc}" >| "${SHELL_RC}"; then
         # Kill all sync jobs and remove sync folder
-        [[ -z ${SKIP_SYNC} ]] && type -a "${SYNC_COMMAND_NAME}" &> /dev/null && {
-            "${SYNC_COMMAND_NAME}" -k all &> /dev/null || :
+        [ -z "${SKIP_SYNC}" ] && command -v "${SYNC_COMMAND_NAME}" 2> /dev/null 1>&2 && {
+            "${SYNC_COMMAND_NAME}" -k all 2> /dev/null 1>&2 || :
             rm -rf "${INFO_PATH}"/sync "${INSTALL_PATH:?}"/"${SYNC_COMMAND_NAME}"
         }
-        rm -f "${INSTALL_PATH}"/{"${COMMAND_NAME}","${UTILS_FILE}"}
-        rm -f "${INFO_PATH}"/{google-drive-upload.info,google-drive-upload.binpath,google-drive-upload.configpath,update.log}
-        [[ -z $(find "${INFO_PATH}" -type f) ]] && rm -rf "${INFO_PATH}"
+        rm -f "${INSTALL_PATH}"/"${COMMAND_NAME}" "${INSTALL_PATH}"/*utils.${INSTALLATION} \
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5865418)